recipe/meta.yaml

```diff
@@ -31,11 +29,6 @@ outputs:
           - python_abi {{ python_maj_min }} *_pypy{{ python_maj_min.replace(".", "") }}_pp{{ pypy_abi }}
         noarch:
           - {{ pin_subpackage("python", max_pin="x.x", min_pin="x.x") }}
-      script:
-        - ln -sf ${PREFIX}/bin/pypy3 ${PREFIX}/bin/python
-        - ln -sf ${PREFIX}/bin/pypy3 ${PREFIX}/bin/python3
-        - ln -sf ${PREFIX}/bin/pypy3 ${PREFIX}/bin/python{{ python_maj_min }}
```

Comment:
This symlink is not there in `pypy3.7` package right?

Reply:
It was added in response to this repo [here](https://foss.heptapod.net/pypy/pypy/-/blob/release-pypy3.7-v7.3.5/pypy/tool/release/package.py#L252) in [a commit](https://foss.heptapod.net/pypy/pypy/-/commit/bc470c4a1fe5c9e082842e7123369b8aa3866577) six months ago